be/src/exec/exec_node.cpp

```diff
@@ -636,12 +637,14 @@ Status ExecNode::eval_conjuncts(const std::vector<ExprContext*>& ctxs, vectorize
             continue;
         } else if (0 == true_count) {
             // all not hit, return
+            filter->assign(chunk->num_rows(), 0);
```

Comment:
Why need assign here?

Reply:
Otherwise we could not got the expression result from filter state.

Reply:
true_count is 0, the below chunk is shrinked to zero rows, so no need introduce extra operations to harm performance.

Reply:
fixed